### ISSUE TITLE
Get intermediate layer output of tf-hub

### ISSUE THREAD

**Issue by OleNet** [tensorflow/hub#396]
Hello, 
I'm researching on the recently released model [AlBert](https://github.com/google-research/google-research/tree/master/albert) which is released in the form of tf-hub.

According the [paper of AlBert](https://arxiv.org/abs/1909.11942), in the shared parameter mode, the Transformer block is stacked L layers by sharing same Transformer parameters. 

Now my question is how can I get the output of specific operator at specific layer? For example, I want to get the output for the multi-head attention at 3rd layer.

**Comment by 0x0539** [comment]
You can retrieve the output of any op from the TF Graph by calling `graph.get_tensor_by_name('full/name/of/op:0')`.

I'm not sure which op you mean precisely by "output for the multi-head attention".
If you want the transformer block output--the Tensor fed to the subsequent Transformer layer--you can try `bert/encoder/transformer/group_0/layer_0/inner_group_0/LayerNorm_1/batchnorm/add_1:0`

Change `layer_0` to `layer_2` for the third layer.

References:
- API docs: https://www.tensorflow.org/api_docs/python/tf/Graph#get_tensor_by_name
- Similar example for BERT: https://github.com/tensorflow/hub/issues/290
- ALBERT code: https://github.com/google-research/google-research/blob/master/albert/modeling.py#L952

**Comment by Zachary4biz** [comment]
@0x0539  what if I'm not sure about the specific name? Is there a way to present the model structure by operation-name?
